logic/src/commonMain/kotlin/com/wire/kalium/logic/data/publicuser/PublicUserMapper.kt

```diff
@@ -74,7 +77,8 @@ class PublicUserMapperImpl(
         completePicture = userDetailResponse.assets.getCompleteAssetOrNull()
             ?.let { idMapper.toQualifiedAssetId(it.key, userDetailResponse.id.domain) },
         availabilityStatus = UserAvailabilityStatus.NONE,
-        userType = userType
+        userType = userType,
+        botService = userDetailResponse.service?.let { BotService(it.id, it.provider) }
```

Comment:
Just a question to understand more about the context. Does this data comes from this same endpoints as well ? 
And if true, that means Bots are like "users" but with some extra data? 🤔  Did I catch it right?

Reply:
Yup, exactly